lib/carma/client/mule_soft_auth_token_configuration.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module CARMA
+  module Client
+    class MuleSoftAuthTokenConfiguration < Common::Client::Configuration::REST
+      def connection
+        Faraday.new(base_path) do |conn|
+          conn.use :breakers
+          conn.request :instrumentation, name: service_name
+          conn.adapter Faraday.default_adapter
+        end
+      end
+
+      def service_name
+        self.class.name
+      end
+
+      def timeout
+        settings.key?(:timeout) ? settings.timeout : 30
```

Comment:
What should this default be? The existing perform call for the form defaults to 10 but we set it to 600 in the default settings.